src/simplewallet/simplewallet.cpp

```diff
@@ -2921,19 +2908,14 @@ bool simple_wallet::sweep_main(uint64_t below, const std::vector<std::string> &a
   std::vector<std::string> local_args = args_;
 
   size_t fake_outs_count;
-  if(local_args.size() > 0) {
-    size_t ring_size;
-    if(!epee::string_tools::get_xtype_from_string(ring_size, local_args[0]))
-    {
-      fake_outs_count = m_wallet->default_mixin();
-      if (fake_outs_count == 0)
-        fake_outs_count = DEFAULT_MIX;
-    }
-    else
-    {
-      fake_outs_count = ring_size - 1;
-      local_args.erase(local_args.begin());
-    }
+  if (local_args.size() > 0) {
```

Comment:
Could you please change this structure to:

```
fake_outs_count = DEFAULT_MIX;
if (epee::string_tools::get_xtype_from_string(ring_size, local_args[0])) {
    local_args.erase(local_args.begin());
}
```

For better code optimization/performance.